lib/recurly/client.rb

```diff
@@ -126,30 +126,30 @@ def handle_response!(request, faraday_resp)
       response = HTTP::Response.new(faraday_resp, request)
       raise_api_error!(response) unless (200...300).include?(response.status)
       resource = if response.body
```

Comment:
seems that rufo going back and forth about which syntax to use.

Reply:
Saw that and just verified locally that `0.7.0` formats this way, and `0.10.0` is the latest.

I bet if you bumped the version in the `gemspec` then all would be happy again